docs/modules/configure/pages/environment_variables.adoc

```diff
@@ -63,6 +63,12 @@ We try to let most of the things defined through Environment Variables, as https
 |SMTP_USERNAME
 |Username for the SMTP server, for sending emails. See xref:services:smtp.adoc[SMTP]
 
+|VAPID_PUBLIC_KEY
```

Comment:
Event though the [docs](https://github.com/zaru/webpush#with-vapid-1) give instructions on using ENV vars directly, I think that we should follow the same pattern with `config/secrets.yml` to be consistent with the changes from #8725 - I'd like to hear @microstudi opinion on this one for sure

Reply:
Yes I agree, we should follow the principle of the least surprise and have everything as evident as possible, although depending on which one is merged first will force the other one to adapt! :wink:

Reply:
Ok! It's a better way to deal with ENV vars, no doubt. 
So, what's your recommendation? Should we merge this pr to develop and then update it in a new task when the #8725 is ready?, or maybe wait until is merged?

Reply:
We usually merge whatever is first done. Most of the time, I don't like blocking one PR until another is done -- specially  when the PRs are from different teams.